libcalico-go/lib/apiconfig/apiconfig.go

```diff
@@ -48,13 +48,14 @@ type CalicoAPIConfigSpec struct {
 }
 
 type EtcdConfig struct {
-	EtcdEndpoints    string `json:"etcdEndpoints" envconfig:"ETCD_ENDPOINTS"`
-	EtcdDiscoverySrv string `json:"etcdDiscoverySrv" envconfig:"ETCD_DISCOVERY_SRV"`
-	EtcdUsername     string `json:"etcdUsername" envconfig:"ETCD_USERNAME"`
-	EtcdPassword     string `json:"etcdPassword" envconfig:"ETCD_PASSWORD"`
-	EtcdKeyFile      string `json:"etcdKeyFile" envconfig:"ETCD_KEY_FILE"`
-	EtcdCertFile     string `json:"etcdCertFile" envconfig:"ETCD_CERT_FILE"`
-	EtcdCACertFile   string `json:"etcdCACertFile" envconfig:"ETCD_CA_CERT_FILE"`
+	EtcdEndpoints       string `json:"etcdEndpoints" envconfig:"ETCD_ENDPOINTS"`
+	EtcdDiscoverySrv    string `json:"etcdDiscoverySrv" envconfig:"ETCD_DISCOVERY_SRV"`
+	EtcdUsername        string `json:"etcdUsername" envconfig:"ETCD_USERNAME"`
+	EtcdPassword        string `json:"etcdPassword" envconfig:"ETCD_PASSWORD"`
+	EtcdKeyFile         string `json:"etcdKeyFile" envconfig:"ETCD_KEY_FILE"`
+	EtcdCertFile        string `json:"etcdCertFile" envconfig:"ETCD_CERT_FILE"`
+	EtcdCACertFile      string `json:"etcdCACertFile" envconfig:"ETCD_CA_CERT_FILE"`
+	EtcdTLSCipherSuites string `json:"etcdTLSCipherSuites" envconfig:"ETCD_TLS_CIPHER_SUITES"`
```

Comment:
libcalico is imported into all of our other components, so likely having this will mean we need both `TLS_CIPHER_SUITES` and `ETCD_TLS_CIPHER_SUITES` specified in each component.

Can we just drive this off the same environment variable - `TLS_CIPHER_SUITES` - for everything?